### PR TITLE
max l337 h4x0r ability

### PR DESCRIPTION
**Pull request by lucyyu**
increase your h4x0rn355

### CODE REVIEW

**Comment by kpeng94** [comment]
this code is terrible

**Comment by dchen496** [comment]
nope!

File: vimrc [hunk 1]
```diff
@@ -93,6 +93,9 @@ set autoindent
 set noswapfile
 set mouse=a
 
+set cursorcolumn
+set cursorline
+
```
Comment (choochootrain) [inline vimrc:98]: rm whitespace - you're polluting the git blame

File: vimrc [hunk 2]
```diff
@@ -93,6 +93,9 @@ set autoindent
 set noswapfile
 set mouse=a
 
+set cursorcolumn
```
Comment (choochootrain) [inline vimrc:96]: rendering this is linear with your screen size - have you considered not creating such an egregious performance regression?

alternatively, build vim in a docker container using nodejs so users can `brew install apt-get` and then `yum emerge vim --fast` and get your fix

File: vimrc [hunk 3]
```diff
@@ -93,6 +93,9 @@ set autoindent
...
+set cursorline
+
 " built-in TeX indentation is bad
 au BufRead,BufNewFile *.tex setlocal ai nocin nosi inde=
```
Comment (choochootrain) [inline vimrc:100]: unrelated comment about code you didn't touch

**Review by choochootrain — COMMENTED** [review]
refusing to sign off on code because its shit and i think i can do it better

**Pull request closed without merging**